pkgs/os-specific/linux/apparmor/default.nix

```diff
@@ -107,6 +107,11 @@ let
     postPatch = ''
       cd ./libraries/libapparmor
     '';
+    
+    # -flto-partition=none is not recognized by Clang
+    preConfigure = if (stdenv.cc.isClang or false) then ''
```

Comment:
this can just use `lib.optionalString`. also, is the `or false` rellay necessary?